rasa/cli/test.py

```diff
@@ -115,29 +116,54 @@ def run_core_test(args: argparse.Namespace) -> None:
     )
 
 
-def run_nlu_test(args: argparse.Namespace) -> None:
-    """Run NLU tests."""
+async def run_nlu_test_async(
+    config: Optional[Union[Text, List[Text]]],
+    data_path: Text,
+    models_path: Text,
```

Comment:
that can be `None`, no?
```suggestion
    models_path: Optional[Text],
```

Reply:
So I thought I should remove the `Optional` because I saw that it always gets a default value so it will never be `None` as long as it is run through the command line. 
`_StoreAction(option_strings=['-m', '--model'], dest='model', nargs=None, const=None, default='models'`

Reply:
Very good point! That can indeed not be `None`. Sorry for the confusion!